src/comp.py

```diff
@@ -2,9 +2,11 @@
 # the user entered into list x.
 
 x = input("Enter comma-separated numbers: ").split(',')
+print(x)
 
 # What do you need between the square brackets to make it work?
-y = []
+y = [1,2,3,"a", "b", "c", x]
```

Comment:
To make this one work, you should define `y` in terms of _filtering_ the user-supplied values stored in `x`.